### PR TITLE
Remove code repetition of login and whats-new test cases for iOS

### PR DESCRIPTION
**Pull request by bilalawan321**
[LEARNER-8696](https://openedx.atlassian.net/browse/LEARNER-8696)

**Description**

- create a new login smoke page for login and whats-new test cases and inherit all test files with it
- remove login and whats-new test case from all files
- fix failed test cases

### CODE REVIEW

File: tests/ios/tests/test_ios_login.py [hunk 1]
```diff
@@ -210,9 +210,14 @@ def test_landscape_smoke(self, set_capabilities, setup_logging):
...
         ) is False
 
-        assert ios_login_page.login(global_contents.login_user_name, global_contents.login_password, False)
+        ios_login_page.login(global_contents.login_user_name, global_contents.login_password)
         setup_logging.info('{} is successfully logged in'.format(global_contents.login_user_name))
         global_contents.turn_orientation(set_capabilities, global_contents.PORTRAIT_ORIENTATION)
+        ios_main_dashboard_page = IosMainDashboard(set_capabilities, setup_logging)
+        ios_main_dashboard_page.get_drawer_icon().click()
+        assert ios_main_dashboard_page.log_out().text == strings.LOGIN
+        assert ios_main_dashboard_page.load_ios_landing_page(
```
Comment (jawad-khan) [inline tests/ios/tests/test_ios_login.py:219]: can we break such lines?

File: tests/ios/pages/ios_course_dashboard.py [hunk 2]
```diff
@@ -302,6 +302,19 @@ def get_course_resume_row(self):
...
 
+    def get_resources_back_icon(self):
+        """
+        Load course dashboard screen
+
+        Returns:
+            webdriver element: course resume Element
+        """
+
+        return self.global_contents.wait_and_get_element(
```
[the remark this answers is not in the document]
Comment (bilalawan321) [reply tests/ios/pages/ios_course_dashboard.py:313]: fixed

File: tests/ios/pages/ios_login_smoke.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    """
+
+    def test_check_login_smoke(self, login, set_capabilities, setup_logging):
+        """
+        Scenarios:
+            Verify Main Dashboard screen is loaded successfully after successful login
+        """
+
+        global_contents = Globals(setup_logging)
+        ios_whats_new_page = IosWhatsNew(set_capabilities, setup_logging)
```
[the remark this answers is not in the document]
Comment (bilalawan321) [reply tests/ios/pages/ios_login_smoke.py:24]: fixed

File: tests/ios/tests/test_ios_login.py (hunk 1 above)
Comment (bilalawan321) [reply tests/ios/tests/test_ios_login.py:219]: fixed

**Review by jawad-khan — APPROVED** [review]

**Pull request merged**